reco_utils/recommender/lightgbm/lightgbm_utils.py

```diff
@@ -0,0 +1,224 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import os, logging
+from sklearn.metrics import (
+    roc_auc_score,
+    log_loss,
+    mean_squared_error,
+    accuracy_score,
+    f1_score,
+)
+import numpy as np
+import pandas as pd
+import category_encoders as ce
+from tqdm import tqdm
+import collections
+import gc
+
+import tarfile
+from reco_utils.dataset.url_utils import maybe_download
+
+logging.basicConfig(level = logging.INFO, format = '%(asctime)s [INFO] %(message)s')
+
+def download_lgb_resources(azure_container_url, data_path, remote_resource_name):
```

Comment:
would be better to consolidate dataset loading under reco_utils/datasets

Reply:
+1

Reply:
Is it means that I'd better add a new file named "criteo.py" into that dataset directory?

Reply:
Agree with this. in the `criteo.py` we can add something like we have in movielens: https://github.com/Microsoft/Recommenders/blob/master/reco_utils/dataset/movielens.py#L133

I think it will be good to take a look at what @jreynolds01 is doing: he has a function called load_spark_df https://github.com/Microsoft/Recommenders/blob/jeremr-criteo-dataloader/reco_utils/dataset/criteo_dac.py#L23, we can do a function called `load_pandas_df`.

Ideally, we want to do single responsibility, like we were discussing here: https://github.com/Microsoft/Recommenders/pull/575#discussion_r260704511

Also, @motefly, we have a function called `maybe_download` that can be useful to you  https://github.com/Microsoft/Recommenders/blob/master/reco_utils/dataset/url_utils.py#L11

Reply:
I have added the `criteo_dac.py` and the `load_pandas_df` function. Please review it again.

Reply:
this is great. I'm working with @jreynolds01 on the data loader for spark https://github.com/Microsoft/Recommenders/pull/619, when your code is merged I will modify it a little to be able to use both the small and the big criteo